### ISSUE TITLE
Pages missing

### ISSUE THREAD

**Issue by yswg** [Ysurac/FlightAirMap#200]
I've installed FlightAirMap on a VMBOX running Debian 8. It is running on Apache2 and the main page is working fine (aircraft, routes ect) but when I click on one of the top links, I just get a page cannot be found.

Any ideas?

**Comment by Ysurac** [comment]
You need to have mod_rewrite enabled. Check https://github.com/Ysurac/FlightAirMap/wiki/Installation

**Comment by Ysurac** [comment]
You should also check if **$globalURL** in *require/settings.php* is set to the right value.

**Comment by yswg** [comment]
`// GLOBAL URL
$globalURL = '/flightairmap';`

Done "sudo a2enmod rewrite" but states "Module rewrite already enabled"

`DocumentRoot /var/www/html

<Directory />
        Options Indexes FollowSymLinks
        AllowOverride All
</Directory>

<Directory /var/www/html/flightairmap>
        Options Indexes FollowSymLinks MultiViews
        AllowOverride All
        Order allow,deny
        allow from all
</Directory>`

**Comment by Ysurac** [comment]
You shouldn't have *Directory /* but *Directory /var/www/html*.
For *Directory /var/www/html/flightairmap*, MultiViews must be disabled so: **-MultiViews**

If you always have errors, paste the url and the error here.

**Comment by yswg** [comment]
Ok, that helped but the dash **-**MultiViews caused Apache2 the error. 

With Debian I finally worked it out that it is apache2.conf needs to be edited and not 000-default.conf. 

This is what I have and works.

`<Directory /var/www/html>`
    `Options Indexes FollowSymLinks`
    `AllowOverride All`
    `Order allow,deny`
    `Allow from all`
`</Directory>`

Though I've noticed that I'm missing quite a lot of "Airports", need to readd them via MySQL?
